python/ccxt/async_support/base/exchange.py

```diff
@@ -476,6 +476,7 @@ def on_error(self, client, error):
             del self.clients[client.url]
 
     def on_close(self, client, error):
+        self.clear_cache()
```

Comment:
Should we give the user an option to control this? I think some might want to keep it

Also maybe we should add something to the docs explaining how to enable/disable this option